app/views/registrations/_form.html.haml

```diff
@@ -2,7 +2,11 @@
 = simple_form_for([@event, @registration], validate: true) do |f|
   = f.input :started_at, as: :hidden, input_html: { value: @registration.started_at }
   = f.input :step_to_validate, as: :hidden, input_html: { value: "start" }
+<<<<<<< HEAD
+  = f.input, :user_id, as: :hidden, input_html: { value: current_user.try(:id) }
+=======
   = f.input :user_id, as: :hidden, input_html: { value: current_user.try(:id) }
+>>>>>>> 406073c793d5418d3736551e0555d8c5d194e9e3
```

Comment:
@willblake01 these look like merge conflicts. This PR shouldn't include changes to the registrations form.

Reply:
@ColinW520 Oh, that's strange. I think that got back in there when I did the rebase. I'm used to just accepting the incoming change from master when merging. After reading about rebasing it sounds like I need to check the conflicts closer.